snippets/base/migrations/0002_auto_20160215_1300.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import snippets.base.fields
+import snippets.base.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='uploadedfile',
+            name='file',
+            field=models.FileField(upload_to=snippets.base.models._generate_filename),
```

Comment:
While looking through the open PRs against django-storages-redux, I noticed https://github.com/jschneier/django-storages/pull/111 which looks like a promising upgrade.